src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -44,30 +45,41 @@ import { WrappedColumn } from './wrapped-column';
               <ng-content></ng-content>
           </ng-template>
 
-          <button class="datagrid-column-title" *ngIf="sortable" (click)="sort()" type="button">
-              <ng-container *ngTemplateOutlet="columnTitle"></ng-container>
+          <button 
+            class="datagrid-column-title" 
+            [attr.aria-label]="commonStrings.sort"
+            *ngIf="sortable" 
+            (click)="sort()" 
+            type="button">
+              <ng-container  *ngTemplateOutlet="columnTitle"></ng-container>
               <clr-icon
                       *ngIf="sortIcon"
                       [attr.shape]="sortIcon"
                       class="sort-icon"></clr-icon>
           </button>
 
           <span class="datagrid-column-title" *ngIf="!sortable">
-               <ng-container *ngTemplateOutlet="columnTitle"></ng-container>
-            </span>
+              <ng-container *ngTemplateOutlet="columnTitle"></ng-container>
+          </span>
 
           <clr-dg-column-separator></clr-dg-column-separator>
       </div>
   `,
   host: {
     '[class.datagrid-column]': 'true',
     '[attr.aria-sort]': 'ariaSort',
+    '[attr.tabindex]': '-1',
```

Comment:
Why did you add this?

Reply:
This is from one of your comment under #1930 
`Can also set tabindex -1 on the column resize handles`

But maybe I did not get it right?

Reply:
That isn't where you put it, you put it on the column container. I think we can just remove this line with no change anyways.